src/esp_loader.ts

```diff
@@ -57,7 +58,7 @@ export class ESPLoader extends EventTarget {
   _efuses = new Array(4).fill(0);
   _flashsize = 4 * 1024 * 1024;
   debug = false;
-  IS_STUB = false;
+  IS_STUB = true;
```

Comment:
The first time it loads the stub is not loaded. Why set this to true?

Reply:
true (left over from tests)